resources/eds_templates/diff_report/templates/main.html

```diff
@@ -16,75 +16,84 @@
-{#    <h1>Dct identifier</h1>#}
-    {% include "sections/added_property_concept_dct_identifier.html" with context%}
+    {% include "sections/added_property_concept_dct_identifier.html" with context %}
     {% include "sections/deleted_property_concept_dct_identifier.html" with context %}
-{#    <h1>Notation</h1>#}
-{#    <h1>Notation value</h1>#}
 
     <h1>In scheme belonging</h1>
-    {% include "sections/added_property_concept_skos_in_scheme.html" with context%}
-    {% include "sections/deleted_property_concept_skos_in_scheme.html" with context%}
+    {% include "sections/added_property_concept_skos_in_scheme.html" with context %}
+    {% include "sections/deleted_property_concept_skos_in_scheme.html" with context %}
 
     <h1>Definitions and scope notes</h1>
-{#    <h1>SKOS definition</h1>#}
-    {% include "sections/added_property_concept_skos_definition.html" with context%}
-    {% include "sections/deleted_property_concept_skos_definition.html" with context%}
-{#    <h1>Scope note</h1>#}
-    {% include "sections/added_property_concept_scope_note.html" with context%}
-    {% include "sections/deleted_property_concept_scope_note.html" with context%}
+    {% include "sections/notes/added_property_concept_skos_definition.html" with context %}
+    {% include "sections/notes/deleted_property_concept_skos_definition.html" with context %}
+    {% include "sections/notes/added_property_concept_scope_note.html" with context %}
+    {% include "sections/notes/deleted_property_concept_scope_note.html" with context %}
+    {% include "sections/notes/added_property_concept_example.html" with context %}
+    {% include "sections/notes/deleted_property_concept_example.html" with context %}
 
     <h1>Other notes</h1>
-{#    <h1>SKOS change note</h1>#}
-    {% include "sections/added_property_concept_skos_changeNote.html" with context%}
-    {% include "sections/deleted_property_concept_skos_changeNote.html" with context%}
-
-{#    <h1>Editorial notes</h1>#}
-    {% include "sections/added_property_concept_editorialNote.html" with context%}
-    {% include "sections/deleted_property_concept_editorialNote.html" with context%}
-{#    <h1>Example</h1>#}
-    {% include "sections/deleted_property_concept_example.html" with context%}
-    {% include "sections/deleted_property_concept_example.html" with context%}
-{#    <h1>History note</h1>#}
-    {% include "sections/added_property_concept_history_note.html" with context%}
-    {% include "sections/deleted_property_concept_history_note.html" with context%}
-
-    {% include "sections/added_property_concept_skos_note.html" with context%}
-    {% include "sections/deleted_property_concept_skos_note.html" with context%}
-
+    {% include "sections/notes/added_property_concept_skos_change_note.html" with context %}
+    {% include "sections/notes/deleted_property_concept_skos_change_note.html" with context %}
+
+    <h1>Semantic Properties</h1>
```

Comment:
we shall have here *skos:narrower* as well

Reply:
`skos:narrower` is not defined in the `src-ap` document